scripts/set_gem_timing

```diff
@@ -8,54 +8,54 @@ IOC_PATH_SC="ioc/fee/GasDetDAQ/R4.0.22-NOTS"
 IOC_PATH=0
 
 # User input validation
-if [[ -n $1 ]]; then # if the user passed at least one arg
-  if [[ $1 != "$NC_STRING" ]] && [[ $1 != "$SC_STRING" ]]; then #validate argument is either SC or NC
-    echo "Error. Argument must be either: [NC or SC]"
-    exit 1
-  else
-    echo "User passed argument of: $1"
-  fi
+if [[ -n $1 ]]; then                                              # if the user passed at least one arg
+    if [[ $1 != "$NC_STRING" ]] && [[ $1 != "$SC_STRING" ]]; then #validate argument is either SC or NC
+        echo "Error. Argument must be either: [NC or SC]"
+        exit 1
+    else
+        echo "User passed argument of: $1"
+    fi
 else
-  echo "Error. Must pass at least one argument"
-  exit 1
+    echo "Error. Must pass at least one argument"
+    exit 1
 fi
 
 if [[ -n $2 ]]; then
-  echo "Error. Must pass strictly one argument, at least one additional arg: $2, please run again with exclusively [NC or SC] as argument"
-  exit 1
+    echo "Error. Must pass strictly one argument, at least one additional arg: $2, please run again with exclusively [NC or SC] as argument"
+    exit 1
 fi
 
 #get caget and caput
 source /cds/group/pcds/setup/epicsenv-cur.sh #Note: (josh) we should only source the actual executables we are using, this is heavy handed. shellcheck does not love this; however if this path doesnt exist we have bigger problems
 #get imgr
-IMGR_CMD_PATH="/reg/g/pcds/engineering_tools/latest-released/scripts/imgr"
+IMGR_CMD_PATH="/cds/group/pcds/engineering_tools/latest-released/scripts/imgr"
 
 # check current operational state
 OP_MODE=$(caget KFE:CAM:TPR:02:MODE | awk '{print $2}')
 SET_MODE_CMD="caput KFE:CAM:TPR:02:MODE $1"
 
 echo "Current OP_MODE is: $OP_MODE"
-if [[ $OP_MODE = "$1" ]]; then
-  echo "Warning. Already in requested mode: $OP_MODE" #Note: (josh) we are opting to consider this not error worthy
+if [[ $OP_MODE == "$1" ]]; then
```

Comment:
Interesting that I've seen recommendations that `=` be used for other posix shell compatibility, but since this is explicitly a bash script I suppose this is fine/equivalent

Reply:
Using double brackets it also not posix compliant. I do not think anything used at slac would have issues with either any features like this.